workspaces/kubernetes-native-dxp/k8s/remoteappa/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     spec:
       containers:
         - name: remote-app-a
-          image: remoteappa
+          image: remote-app-a
```

Comment:
I've been meaning to unify the difference between the image and the service. For example, the service is called `liferay-hello-world` and then we have variations of `remoteappa`, `remote-app-a` elsewhere. I'd like to migrate to simply `liferay-hello-world` everywhere since that's what it is.

However, I think we need to ensure we don't break underlying LCP assumptions about names of services. I seem to recall something about not liking dashes in service names. If that is the case, we have lots of renaming to do.

Reply:
Yeah the naming we will need to normalize.  should we drop the `-`?  Because I do believe LCP has trouble.

Reply:
should be relatively easy to test once we're pushing projects to a SANDBOX.

Reply:
On the naming, I agree, we have it named "by form" in some places, and "by function" in another place.  I guess we should keep it "by function" for the demo?  So `liferay-hello-world` instead of `remoate-app-a`?

Reply:
Ok i'll defer the renaming until another PR